pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -487,18 +486,19 @@ func (l *listUsersQuery) expandExclusion(
 	baseFoundUsersCh := make(chan *openfgav1.User, 1)
 	subtractFoundUsersCh := make(chan *openfgav1.User, 1)
 
-	var errs error
+	var errBase error
+	var errSub error
```

Comment:
```suggestion
	var baseError, substractError error
```

Just a suggestion, I prefer more descriptive var naming.